de.metas.adempiere.libero.libero/src/main/java/org/eevolution/model/validator/PP_Product_BOMLine.java

```diff
@@ -127,4 +114,14 @@ public void updateProductLowestLevelCode(final I_PP_Product_BOMLine bomLine)
 
 		// TODO: if product's low level changed, we need to update the low level from all bom components where this product is in bom header.
 	}
+
+	@CalloutMethod(columnNames = { I_PP_Product_BOMLine.COLUMNNAME_VariantGroup})
+	public void validateVariantGroup(final I_PP_Product_BOMLine bomLine)
+	{
+		final boolean valid = Services.get(IProductBOMBL.class).isValidVariantGroup(bomLine);
+		if (!valid)
+		{
+			throw new LiberoException("@NoSuchVariantGroup@");
```

Comment:
Maybe we should extend/translate the error message? 
(not sure because i'm not sure what a variant group actually is)